web/src/app/shared/dialog/markdown/markdown.component.ts

```diff
@@ -0,0 +1,47 @@
+// Core modules
+import { Component, OnInit, ViewEncapsulation } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+// Third party modules
+import { HttpClient } from '@angular/common/http';
+
+// Application models
+import { Help } from '@app/main/components/help/help';
+import { HelpModel } from '@shared/models/index.model';
+
+@Component({
+  selector: 'dashboard-markdown',
+  templateUrl: './markdown.component.html',
+  styleUrls: ['./markdown.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+})
+export class MarkdownComponent implements OnInit {
```

Comment:
`MarkdownComponent` implies it is a generic component but it is specific to `help`, `HelpComponent` I think would make more sense?

Reply:
HelpComponent is already there . What should be the name for this component ?
I have just renamed the dialog-markdown.component to markdown.component

Reply:
May be, `HelpDetailComponent`?

Reply:
Done